var/spack/repos/builtin/packages/py-jsonlines/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyJsonlines(PythonPackage):
+    """Library with helpers for the jsonlines file format"""
+
+    homepage = "https://github.com/wbolster/jsonlines"
+    pypi = "jsonlines/jsonlines-4.0.0.tar.gz"
+
+    license("BSD-3-Clause", checked_by="alex391")
```

Comment:
Can you verify the license and add your username instead? Or we would also welcome if @alex391 can confirm here in this thread.

Reply:
Nevermind, I see the authorship on the commits now.